demo/agilebank/templates/k8srequiredprobes_template.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8srequiredprobes
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sRequiredProbes
+      validation:
+        openAPIV3Schema:
+          properties:
+            message:
+              type: string
+
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package k8srequiredprobes
+
+        get_constraint_message(parameters) = msg {
+          msg := parameters.message
+        }
+
+        violation[{"msg": msg}] {
```

Comment:
Does the violation -> probe_violation indirection give us anything, or can we streamline this?

Reply:
Thank you for your review and feedback. I'll do some cleanup per your suggestions.